opentutor_classifier/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool]
+[tool.poetry]
+name = "opentutor_classifier"
+version = "1.0.0"
+description="train and run inference for open tutor"
+authors=[]
+
+[tool.poetry.dependencies]
+# python = "^3.8"
+python = ">=3.8,<3.10"
```

Comment:
in practice, we need to lock this project down to a specific version of python at least to the minor version level (currently 3.8)...do we need this allowance for 3.9, 3.10?